core/src/test/java/io/aiven/klaw/dao/migration/MigrateData2x5x0Test.java

```diff
@@ -0,0 +1,90 @@
+
+  @Mock private SelectDataJdbc selectDataJdbc;
+
+  @Mock private InsertDataJdbc insertDataJdbc;
+
+  private UtilMethods utilMethods;
+
+  @BeforeEach
+  public void setUp() {
+    migrateData2x5x0 = new MigrateData2x5x0(selectDataJdbc, insertDataJdbc);
+    utilMethods = new UtilMethods();
+  }
+
+  @Test
+  public void addSequencesWhenNoData() {
+    when(selectDataJdbc.getDataFromKwEntitySequences()).thenReturn(0L);
+    when(selectDataJdbc.getTenants()).thenReturn(utilMethods.getTenants());
+    when(selectDataJdbc.getNextClusterId(anyInt())).thenReturn(null);
+    when(selectDataJdbc.getNextEnvId(anyInt())).thenReturn(null);
+    when(selectDataJdbc.getNextTeamId(anyInt())).thenReturn(null);
+
+    boolean success = migrateData2x5x0.migrate();
+    verify(insertDataJdbc, times(1))
+        .insertIntoKwEntitySequence(eq(EntityType.CLUSTER.name()), eq(101), eq(101));
+    verify(insertDataJdbc, times(1))
+        .insertIntoKwEntitySequence(eq(EntityType.ENVIRONMENT.name()), eq(101), eq(101));
+    verify(insertDataJdbc, times(1))
+        .insertIntoKwEntitySequence(eq(EntityType.TEAM.name()), eq(101), eq(101));
+    assertThat(success).isTrue();
+  }
+
+  @Test
+  public void addSequencesWhenDataAvailable() {
+    when(selectDataJdbc.getDataFromKwEntitySequences()).thenReturn(0L);
+    when(selectDataJdbc.getTenants()).thenReturn(utilMethods.getTenants());
+    when(selectDataJdbc.getNextClusterId(anyInt())).thenReturn(101);
+    when(selectDataJdbc.getNextEnvId(anyInt())).thenReturn(102);
+    when(selectDataJdbc.getNextTeamId(anyInt())).thenReturn(1003);
+
+    boolean success = migrateData2x5x0.migrate();
+    verify(insertDataJdbc, times(1))
+        .insertIntoKwEntitySequence(eq(EntityType.CLUSTER.name()), eq(102), eq(101));
+    verify(insertDataJdbc, times(1))
+        .insertIntoKwEntitySequence(eq(EntityType.ENVIRONMENT.name()), eq(103), eq(101));
+    verify(insertDataJdbc, times(1))
+        .insertIntoKwEntitySequence(eq(EntityType.TEAM.name()), eq(1004), eq(101));
+    assertThat(success).isTrue();
+  }
+
+  @Test
+  public void addSequencesWhenSequencesUpdatedAlready() {
+    when(selectDataJdbc.getDataFromKwEntitySequences()).thenReturn(3L);
+    when(selectDataJdbc.getTenants()).thenReturn(utilMethods.getTenants());
+    when(selectDataJdbc.getNextClusterId(anyInt())).thenReturn(101);
+    when(selectDataJdbc.getNextEnvId(anyInt())).thenReturn(102);
+    when(selectDataJdbc.getNextTeamId(anyInt())).thenReturn(1003);
+
+    boolean success = migrateData2x5x0.migrate();
+    verify(insertDataJdbc, times(0))
+        .insertIntoKwEntitySequence(eq(EntityType.CLUSTER.name()), eq(102), eq(101));
+    verify(insertDataJdbc, times(0))
+        .insertIntoKwEntitySequence(eq(EntityType.ENVIRONMENT.name()), eq(103), eq(101));
+    verify(insertDataJdbc, times(0))
+        .insertIntoKwEntitySequence(eq(EntityType.TEAM.name()), eq(1004), eq(101));
+    assertThat(success).isTrue();
+  }
```

Comment:
Maybe add a test where the update failed half way through and one for multiple tenants.